lib/Default/SmrPlayer.class.inc

```diff
@@ -1543,6 +1543,22 @@ class SmrPlayer extends AbstractSmrPlayer {
 		}
 	}
 
+	// Get bounties that can be claimed by this player
+	// Type must be 'HQ' or 'UG'
+	public function getClaimableBounties($type) {
+		$bounties = array();
```

Comment:
This can be changed to:
```php
return array_filter($this->getBounties(), function($bounty) use  ($type) {
	return $bounty['Type'] === $type;
});

Reply:
Actually, the reason this function is needed as-is is because the many existing bounties methods `getBounties`, etc. based on `getBountiesData`) use `account_id=$this->getAccountID()`, whereas `getClaimableBounties` needs `claimer_id=$this->getAccountID()`.

Perhaps the method name needs to be clearer? (maybe `getBountiesClaimableByPlayer`?)

Reply:
No, it makes sense, I just didn't click the difference